mabl snippets/insertText.js

```diff
@@ -1,11 +1,11 @@
 /**
  * Run a small snippet of JavaScript during a mabl flow/journey
```

Comment:
You didn't change this, but if you have time can you update the description here to describe the snippet?